webapi/_test_data/ontologies/beol-onto.ttl

```diff
@@ -922,6 +1465,16 @@
                             owl:minCardinality "0"^^xsd:nonNegativeInteger ;
```

Comment:
Shouldn't this be `owl:maxCardinality "1"^^xsd:nonNegativeInteger`? Or may a letter have more than one translation?

Reply:
a letter may have more than one translation, it is not the case for Goldbach volume, but might be the case for other leoo volumes

Reply:
Ok, then this is fine :-)